spec/features/orders/show_spec.rb

```diff
@@ -62,27 +62,119 @@
       end
     end
 
-    it "I see a link to cancel the order" do
+    it 'I see a link to cancel the order' do
       visit profile_orders_show_path(@order_1.id)
-      expect(page).to have_link("Cancel Order")
+      expect(page).to have_link('Cancel Order')
     end
 
     xit "When I click the cancel order button, each row in the 'order items' table status is unfulfilled,
         the order is cancelled, items are returned to their merchants, I'm returned to my profile
         page and see flash saying order is cancelled and this order's status is cancelled" do
-        visit profile_orders_show_path(@order_1.id)
+      visit profile_orders_show_path(@order_1.id)
+
+      # Items returned to merchants... Only when "packaged" status?
```

Comment:
I was wondering the same here... At which point are we supposed to actually remove the items from a merchant's inventory?  If during 'packaged' status, I believe we will have to take another look at US 30